### PR TITLE
[Doc] Submit Argo Workflow from SQLFlow Container

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Part of https://github.com/sql-machine-learning/sqlflow/issues/1066

### CODE REVIEW

File: scripts/build_docker_image.sh [hunk 1]
```diff
@@ -129,3 +129,7 @@ git checkout eb93e2a48e6fe8f077c4937d8c0c5987faa9cf56
 pip install -r elasticdl/requirements.txt
 python setup.py install
 cd ..
+
+# 10. Install Argo. 34M total.
```
Comment (Yancey0623) [inline scripts/build_docker_image.sh]: Great job! Do you want to set up Argo on the Minikube in a bash script?

Comment (wangkuiyi) [reply scripts/build_docker_image.sh]: In practice, we do NOT need the `argo` command, so we don't need to edit the Dockerfile.  Instead, it is the submitter program running in the SQLFlow server container who submits Argo workflows.  It does so by calling the Kubernetes Restful API, but not the `argo` command.

Therefore, I would suggest canceling the edits to Dockefile in this PR.

Another note is that the SQLFlow server container is likely running on the same Kubernetes cluster, on which, we run Argo workflows as well.  Calling Kubernetes APIs from a container running on the Kubernetes cluster is known as *Kubernetes-native*.  For how to implement Kubernetes-native calls, please refer to the ElasticDL master program.

File: doc/argo-setup.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Submit Argo Workflow from SQLFlow Container
+
+In this document, we explain how to submit jobs from a SQLFlow server container to a Kubernetes clluster.  We use Minikube on a Mac, but you can use Kubernetes clusters on public cloud services as well.  The jobs we submit in this document are Argo workflows.
```
Comment (shendiaomo) [inline doc/argo-setup.md]: clluster -> cluster

**Review by wangkuiyi — DISMISSED** [review]

File: scripts/build_docker_image.sh (hunk 1 above)
Comment (tonyyang-svail) [reply scripts/build_docker_image.sh]: Hi @wangkuiyi, could you please provide the link to the code snippet in the EDL master program that invokes Kubernetes-native calls? It is hard for me to find it by going through the source code.

Comment (wangkuiyi) [reply scripts/build_docker_image.sh]: https://github.com/sql-machine-learning/elasticdl/blob/develop/elasticdl/python/master/k8s_worker_manager.py#L61

Comment (tonyyang-svail) [reply scripts/build_docker_image.sh]: Removed Argo in Docker image.

**Review by Yancey0623 — APPROVED** [review]
LGTM.

**Pull request merged**